ui/src/logic/beaconchaApi/buildValidatorSummaryURL.ts

```diff
@@ -0,0 +1,23 @@
+import { BeaconchaGetResponse } from "./types";
+import { beaconchaApiParamsMap } from "../../params";
+
+export default function buildValidatorSummaryURL({
+  allValidatorsInfo,
+  network,
+}: {
+  allValidatorsInfo: BeaconchaGetResponse[];
+  network: string;
+}): string {
+  let summaryValidatorURL =
+    beaconchaApiParamsMap.get(network)!.baseUrl + "/dashboard?validators="; //TODO Be careful with the !
+
+  allValidatorsInfo.forEach((validatorInfo) => {
+    summaryValidatorURL += validatorInfo.data.validatorindex + ",";
+  });
+
+  summaryValidatorURL = summaryValidatorURL.slice(0, -1);
```

Comment:
Looks too complicated the way you build the url, can you find another way? also would be a good practice to implement a unit test for this function

Reply:
Would yo prefer to pass "baseUrl" as argument? I don`t see many ways to simplify this